packages/headless/src/controllers/commerce/core/parameter-manager/headless-core-parameter-manager.ssr.ts

```diff
@@ -0,0 +1,104 @@
+import {SSRCommerceEngine} from '../../../../app/commerce-engine/commerce-engine.ssr';
```

Comment:
Not implemented in samples yet.

I created a separate Jira for this because we'll also need to work on a serializer ultil: https://coveord.atlassian.net/browse/KIT-3462